src/Project.hs

```diff
@@ -279,6 +304,13 @@ isInProgress pr = case approvedBy pr of
       BuildSucceeded  -> False
       BuildFailed     -> False
 
+-- Return whether the given commit is, or in this approval cycle ever was, an
+-- integration candidate of this pull request.
+wasIntegrationCandidateFor :: Sha -> PullRequest -> Bool
```

Comment:
Terminology: Here, 'integration candidate' is a value of type `Sha`. In the `ProjectState` record, there is a field `integrationCandidate :: Maybe PullRequestId`.

That was confusing to me. Can we use different terminology for the PR we're attempting to merge/rebase and it's `HEAD`?

Reply:
Hmm, that is a fair point. I’ll rename it to `wasIntegrationResultFor`.

Reply:
Thanks! :)